src/object/stat.js

```diff
@@ -80,6 +80,30 @@ module.exports = (createCommon, options) => {
 
       expect(expected).to.deep.equal(stats)
     })
+    
+    it('should pass on the opts', (done) => {
```

Comment:
We're specifically testing that `timeout` is passed:

```suggestion
    it('should respect timeout option', (done) => {
```